src/extension.ts

```diff
@@ -99,4 +102,24 @@ class ExtensionRuntime {
   public async dispose(): Promise<void> {
     await this.client?.stop();
   }
+
+  public async startClientAndWaitForVersion() {
+    this.client = this.client ?? await DafnyLanguageClient.create(this.context, this.statusOutput);
```

Comment:
Since we recycle this.client if it exists here, shouldn't we also exit right away if we already have a client?

Reply:
I tested this hypothesis and restarting the server no longer works.
I think we still need to start the client - which will start the language server.